<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>983</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>821</width>
      <height>941</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="tabStopWidth">
     <number>40</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHighlighting">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>109</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Highlighting</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_file"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_file"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionChange_font"/>
    <addaction name="actionChange_font_color"/>
    <addaction name="actionChange_font_size"/>
    <addaction name="actionChange_background_color"/>
    <addaction name="separator"/>
    <addaction name="actionSet_tab_stop"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>Compile</string>
    </property>
    <addaction name="actionCompile_gpp"/>
   </widget>
   <addaction name="menuCreate"/>
   <addaction name="menuFont"/>
   <addaction name="menuCompile"/>
  </widget>
  <action name="actionCreate_file">
   <property name="text">
    <string>Create file</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionChange_font_color">
   <property name="text">
    <string>Change font color</string>
   </property>
  </action>
  <action name="actionChange_font_size">
   <property name="text">
    <string>Change font size</string>
   </property>
  </action>
  <action name="actionChange_background_color">
   <property name="text">
    <string>Change background color</string>
   </property>
  </action>
  <action name="actiongpp">
   <property name="text">
    <string>g++</string>
   </property>
  </action>
  <action name="actionSet_tab_stop">
   <property name="text">
    <string>Set tab stop</string>
   </property>
  </action>
  <action name="actionHighlite">
   <property name="text">
    <string>Highlite</string>
   </property>
  </action>
  <action name="actionChange_font">
   <property name="text">
    <string>Change font</string>
   </property>
  </action>
  <action name="actionCompile_gpp">
   <property name="text">
    <string>Compile gpp</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
